content/docs/user-guide/packaging/asset-bundler/_index.md

```diff
@@ -6,14 +6,12 @@ title: Bundle game assets in Open 3D Engine
 
 Open 3D Engine provides a set of tools for intelligently bundling assets and managing your game's product and source dependencies. Use these tools to package only the assets currently used in your game project and reduce the deployment size and complexity of your product overall.
 
+{{< preview-migrated >}}
```

Comment:
Mike just got rid of a lot of preview tags. I thought we weren't using them any longer? Or is this an example of a continued use case?

Reply:
No idea - @micronAMZN added you as a reviewer so that you can tell me if I need to drop these.